src/components/ChatBox.jsx

```diff
@@ -0,0 +1,171 @@
+import React, { useEffect, useState, useRef, useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { io } from 'socket.io-client';
+import {
+  getActiveMeeting,
+  addMessageActiveMeeting,
+} from '../redux/slices/meetingSlice';
+
+const socket = io.connect('http://localhost:3333');
```

Comment:
Put this url in an env variable and have something on the page that says
`SOCKET_URL = {ENV_VARIABLE_NAME} || localhost...`